test/webdriverio/test/actions_test.ts

```diff
@@ -51,13 +79,60 @@ suite('Menus test', function () {
     await this.browser.keys([Key.Ctrl, Key.Return]);
     await this.browser.pause(PAUSE_TIME);
 
-    chai.assert.isTrue(
-      await contextMenuExists(this.browser, 'Help'),
-      'The menu should be openable on a block in the toolbox',
+    chai.assert.deepEqual(
+      process.platform === 'darwin'
+        ? [
+            {'text': 'Help'},
+            {'disabled': true, 'text': 'Move Block M'},
+            {'disabled': true, 'text': 'Cut ⌘ X'},
```

Comment:
Probably a topic for a separate issue / PR, but: shouldn't this be `'⌘X'`, not `'⌘ X'`?  Or is the spurious space just an artefact of how the displayed HTML is being converted to text?

Reply:
I think there is a real space in these context menus. There is some spacing in Mac native menus... perhaps less than a full space?

![image](https://github.com/user-attachments/assets/42b121f1-6558-47ec-a00e-84eb541eb7b8)